packages/flutter/lib/src/widgets/app.dart

```diff
@@ -555,11 +559,18 @@ class WidgetsApp extends StatefulWidget {
   /// This list must not null. Its default value is just
   /// `[const Locale('en', 'US')]`.
   ///
-  /// The order of the list matters. By default, if the device's locale doesn't
-  /// exactly match a locale in [supportedLocales] then the first locale in
-  /// [supportedLocales] with a matching [Locale.languageCode] is used. If that
-  /// fails then the first locale in [supportedLocales] is used. The default
-  /// locale resolution algorithm can be overridden with [localeResolutionCallback].
+  /// The order of the list matters. The default locale resolution algorithm, 
+  /// [basicLocaleListResolution], uses a variation of the truncation algorithm where
+  /// it attempts to match by the following priority: all subtags, language+script,
```

Comment:
I don't think most developers will immediately know what you mean by "all subtags" here. I assume you just mean that the locale's language+country+script all match. And why not refer to the fields with [Locale.language] etc?

Presenting the matches as a list (maybe a numbered list because priority) would make this easier to parse.